src/main/java/frc/robot/commands/GripObject.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Gripper;
+
+public class GripObject extends CommandBase{
+    
+    private boolean isCone;
```

Comment:
The isCone boolean shouldn't be necessary with our current gripper design. Also, the way this class is currently implemented, isCone won't be updated ever. I would recommend just removing the isCone references. If we decide this class needs knowledge of the object type, we can look back at this then.

Reply:
Removed Gripper Command file anyways